runFastDPH5.py

```diff
@@ -55,3 +58,23 @@
   comm_s = f"ssh -q {dimpleNode} \"{os.environ['MXPROCESSINGSCRIPTSDIR']}dimple.sh {request_id} {numstart}\""  
   logger.info(f"running dimple: {comm_s}")
   os.system(comm_s)
+
+if runAutoProc:
+  setBlConfig("auto_proc_lock", True)
+  try:
+    queue = getBlConfig("auto_proc_queue")
+    queue.append((directory, request_id))
+    setBlConfig("auto_proc_queue", queue)
+  except Exception as e:
+    logger.exception("Could not add request to autoproc queue")
+  finally:
+    setBlConfig("auto_proc_lock", False)
+  start_in_proc = None
+  for proc_num in ['uranus-cpu044', 'uranus-cpu041', 'uranus-cpu021']:
+    if not getBlConfig(proc_num):
```

Comment:
Looks like the BlConfig flag for `proc_num` is set, then cleared in the autoproc.py file in lsdc_processing.

All the more reason to make sure the changes related to autoproc are put into a PR for lsdc_processing!